handlers/handlers.go

```diff
@@ -159,6 +159,13 @@ func VersionsList(dc DatasetClient, rend RenderClient, cfg config.Config) http.H
 	})
 }
 
+// DatasetPage will load a zebedee dataset page
+func DatasetPage(zc ZebedeeClient, dc DatasetClient, rend RenderClient, cfg config.Config, apiRouterVersion string) http.HandlerFunc {
```

Comment:
Consider creating separate code files for each handler. This file is getting pretty large, and it might be nicer to have the code / tests for handlers in separate files.

Reply:
So...I had a go at this and it went bad because I wasn't sure what I needed to import and what I could leave out, could we pair on this?